guides/common/modules/proc_setting-an-alternative-directory-for-remote-execution-jobs-in-pull-mode.adoc

```diff
@@ -0,0 +1,28 @@
+[id="setting-an-alternative-directory-for-remote-execution-jobs-in-pull-mode_{context}"]
+= Setting an alternative directory for remote execution jobs in pull mode
+
+.Procedure
+. On your host, create a new directory:
```

Comment:
Why would a user ever want this? It would be good to describe at least one use case.

Reply:
Good question, I will try to figure out some usecase. If I don't find any, I might close this.